pkg/api/types.go

```diff
@@ -1202,6 +1202,8 @@ const (
 	ResourceMemory ResourceName = "memory"
 	// Volume size, in bytes (e,g. 5Gi = 5GiB = 5 * 1024 * 1024 * 1024)
 	ResourceStorage ResourceName = "storage"
+	// Number of Pods that may be running on this Node.
+	ResourceIPs ResourceName = "pods"
```

Comment:
Is this about max pods or specifically about IPs?  If it is about IPs, then use that name everywhere.  Some pods don't need to consume IPs?